app/src/main/res/values/dimens.xml

```diff
@@ -10,6 +10,7 @@
 
     <!--    padding-->
     <dimen name="tiny_padding">7dp</dimen>
+    <dimen name="search_view_padding">8dp</dimen>
```

Comment:
move this padding to another section like specific padding, because of these paddings are common